exercises/exercise5.js

```diff
@@ -0,0 +1,12 @@
+// function to convert a number to its binary representation
+
+const convertNumberToBinary = (num) => {
+  return num.toString(2);
```

Comment:
The returned value when the parameter value is a negative number is not realistic.
eg: toBinary(-7) // -111
**N.B**: In binary number sign are represented as follow:

- 1 is added in front if it is a negative number
- 0 is added in front if it is a positive number